website/source/docs/builders/virtualbox-iso.html.md

```diff
@@ -55,11 +55,11 @@ builder.
 ### Required:
 
 -   `iso_checksum` (string) - The checksum for the OS ISO file. Because ISO
-    files are so large, this is required and Packer will verify it prior to
-    booting a virtual machine with the ISO attached. The type of the checksum is
-    specified with `iso_checksum_type`, documented below. At least one of
-    `iso_checksum` and `iso_checksum_url` must be defined. This has precedence
-    over `iso_checksum_url` type.
+    files are so large, it is highly recommended that you provide a value for
```

Comment:
I believe this is the line that makes checksumming not a required field:
https://github.com/mitchellh/packer/blob/4a00ffeea2206847c5368749c9fc2b879227d2e8/common/iso_config.go#L44

[The docs currently read](https://www.packer.io/docs/builders/virtualbox-iso.html#iso_checksum_type):

> `iso_checksum_type` (string) - The type of the checksum specified in `iso_checksum`. Valid values are "none", "md5", "sha1", "sha256", or "sha512" currently. While "none" will skip checksumming, this is not recommended since ISO files are generally large and corruption does happen from time to time.